### PR TITLE
WIP: Minor bugfixes

### PR DESCRIPTION
**Pull request by jlsnow301**
## About The Pull Request
Cleaning up some errors that have come my way. Things I've noticed:

a) Medical objective greentexting even if it's full of bodies
b) Wanted just a bit added to clone time calculation
c) Changeling free greentext: "Impersonate x person, leave real person behind" is selecting lings to impersonate themselves (sorry Zesko)
d) Mech cable layer removed from unlockable designs, doesn't exist

## Why It's Good For The Game
bruh.wav
Some of these are just touch ups that might not even be noticed 

## Changelog
:cl:
fix: Reworked objective's find_job method to be a bit less redundant, hopefully that works
tweak: Clone time slight increase
fix: Medical objectives coded into list
fix: mech_cable_layer reference removed
/:cl:

### CODE REVIEW

File: code/modules/crew_objectives/medical_objectives.dm [hunk 1]
```diff
@@ -5,26 +5,12 @@
 	jobs = "chiefmedicalofficer,geneticist,medicaldoctor"
 
 /datum/objective/crew/morgue/check_completion()
+	var/list/medical_areas = list(typesof(/area/medical/cryo), typesof(/area/medical/genetics/cloning), typesof(/area/medical/exam_room),
```
Comment (qwertyquerty) [inline code/modules/crew_objectives/medical_objectives.dm]: Correct me if I'm wrong but wont this just make a bunch of nested lists

Comment (jlsnow301) [reply code/modules/crew_objectives/medical_objectives.dm]: I'd like to convert it to a list like list(typesof(/area/x, /area/y,)) but I'm not sure if this works yet

Comment (ike709) [reply code/modules/crew_objectives/medical_objectives.dm]: You could have two lists.
One list of the areas you want to get all the types of, which you then loop through to populate the other list which will have all of the areas you care about.

Comment (jlsnow301) [reply code/modules/crew_objectives/medical_objectives.dm]: in other places this is done by var/x = typecacheof(list(
would that work here?

**Review by qwertyquerty — APPROVED** [review]
Looks good to me

**Pull request merged**